src/query/settings/src/lib.rs

```diff
@@ -364,6 +364,16 @@ impl Settings {
                 desc: "The maximum query execution time. it means no limit if the value is zero. default value: 0.",
                 possible_values: None,
             },
+            SettingValue {
+                default_value: UserSettingValue::String("binary".to_owned()),
+                user_setting: UserSetting::create(
+                    "collation",
+                    UserSettingValue::String("binary".to_owned()),
+                ),
+                level: ScopeLevel::Session,
+                desc: "Char collation, support \"binary\" \"utf8\" default value: binary",
+                possible_values: Some(vec!["binary", "utf8"]),
```

Comment:
why “binary” is the default value not utf-8？

Reply:
Because the string column is stored as binary by default.

Reply:
You can set global to make it default to use utf8